VERSION

```diff
@@ -1 +1 @@
-2.0
+0.0.4
```

Comment:
Did you mean to do this? I thought we wanted all packages to have version 2.0

Reply:
John for scdl + bionemo-core we're going to use something like 0.0.4 and 0.0.1. Homegeneous versioning across all of bionemo is a TBD topic -- what Polina is doing here is a reversible workaround to fix SCDL (which has end users).

Here's the longer discussion: https://nvidia.slack.com/archives/C074Z808N05/p1731011971074089

and an SRD: https://docs.google.com/document/d/1Ug5fYdeo5agV3ywyO4rlgW3DDj53K8c5bgAnYNQsYK8/edit?tab=t.0#heading=h.r83g8cjg4d1y 

but that's all WIP and in the meantime SCDL needs to work.

Reply:
fixed for now.